source/constructs/api/data_source/service.py

```diff
@@ -361,13 +384,341 @@ def sync_jdbc_connection(
+                                    'DeleteBehavior': 'DELETE_FROM_DATABASE'
+                                }
+                            )
+                            logger.info("update rds crawler:")
+                            logger.info(up_cr_response)
+                    except Exception as e:
+                        logger.info("update_crawler error")
+                        logger.info(str(e))
+                    st_cr_response = glue.start_crawler(
+                        Name=crawler_name
+                    )
+                    logger.info(st_cr_response)
+                except Exception as e:
+                    logger.info("sync_rds_connection get_crawler and create:")
+                    logger.info(str(e))
+                    response = glue.create_crawler(
+                        Name=crawler_name,
+                        Role=crawler_role_arn,
+                        DatabaseName=glue_database_name,
+                        Targets={
+                            'JdbcTargets': jdbc_targets,
+                        },
+                        Tags={
+                            'AdminAccountId': _admin_account_id
+                        }
+                    )
+                    logger.info(response)
+                    start_response = glue.start_crawler(
+                        Name=crawler_name
+                    )
+                    logger.info(start_response)
+                crud.create_jdbc_connection(jdbc.account_provider,
+                                            jdbc.account_id,
+                                            jdbc.region,
+                                            jdbc.instance,
+                                            glue_connection_name,
+                                            glue_database_name,
+                                            None,
+                                            crawler_name)
+            else:
+                crud.set_jdbc_connection_glue_state(jdbc.account_provider, jdbc.account_id, jdbc.region, jdbc.instance,
+                                                    MessageEnum.SOURCE_JDBC_NO_SCHEMA.get_msg())
+                raise BizException(MessageEnum.SOURCE_JDBC_NO_SCHEMA.get_code(), MessageEnum.SOURCE_JDBC_NO_SCHEMA.get_msg())
+    except Exception as err:
+        crud.set_jdbc_connection_glue_state(jdbc.account_provider, jdbc.account_id, jdbc.region, jdbc.instance, str(err))
+        glue = boto3.client('glue',
+                            aws_access_key_id=credentials['AccessKeyId'],
+                            aws_secret_access_key=credentials['SecretAccessKey'],
+                            aws_session_token=credentials['SessionToken'],
+                            region_name=jdbc.region
+                            )
+        """ :type : pyboto3.glue """
+        try:
+            glue.delete_crawler(crawler_name)
+        except Exception:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears that you are generically passing an `Exception` object without performing any other operation on it. This may hide error conditions that can otherwise be quickly detected and addressed. We recommend that you catch a more specific exception. If the code must broadly catch all exceptions, consider logging the stack trace using the [logging.exception()](https://docs.python.org/3/library/logging.html#logging.exception) API. For example,

```
try:
    x = 1 / 0
except ZeroDivisionError as e:
    logging.exception(&#39;ZeroDivisionError: %s&#39;, e)

```

Similar issue at line numbers 633 and 637.